### PR TITLE
docs: improve style in Creating & Modifying pages

### PR DESCRIPTION
**Pull request by dandv**
Cosmetic changes:

* italics for Graph<em>i</em>QL, fixing the raw rendering

image

* we -> you style change
* clarify which React components become pages

### CODE REVIEW

**Comment by lannonbr** [comment]
@dandv I'd recommend not requesting reviews from particular core team members in the future unless you've been working with them on said issue / PR.

**Comment by dandv** [comment]
Ok. Sorry about that. Kyle was suggested by GitHub.

File: docs/docs/creating-and-modifying-pages.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Pages can be created in three ways:
 
 - In your site's gatsby-node.js by implementing the API
   [`createPages`](/docs/node-apis/#createPages)
-- Gatsby core automatically turns React components in `src/pages` into pages
+- Gatsby core automatically turns React components exported `default` in `src/pages/*` into pages
```
Comment (laurieontech) [inline docs/docs/creating-and-modifying-pages.md:11]:
```suggestion
- Gatsby core automatically turns React components in `src/pages/*` into pages. Note that you must make the component the default export
```

File: docs/docs/creating-and-modifying-pages.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Pages can be created in three ways:
 
 - In your site's gatsby-node.js by implementing the API
   [`createPages`](/docs/node-apis/#createPages)
-- Gatsby core automatically turns React components in `src/pages` into pages
+- Gatsby core automatically turns React components in `src/pages/*` into pages. Note that you must make the component the default export
```
Comment (marcysutton) [inline docs/docs/creating-and-modifying-pages.md:11]: Some folks reading this page might not know what default exports are exactly, so we shouldn't assume. Linking to more information would help.
```suggestion
- Gatsby core automatically turns React components in `src/pages/*` into pages. Note that you must make the component the [default export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)
```

**Pull request merged**